Boards.h

```diff
@@ -260,6 +260,23 @@ writePort(port, value, bitmask):  Write an 8 bit port.
 #define PIN_TO_SERVO(p)         (p) // deprecated since v2.4
 
 
+// Arduino MKRZero
+#elif defined(ARDUINO_SAMD_MKRZERO)
+#define TOTAL_ANALOG_PINS       7
+#define TOTAL_PINS              34 // 8 digital + 3 spi + 2 i2c + 2 uart + 7 analog + 3 usb + 1 aref + 5 sd + 1 bottom pad + 1 led + 1 battery adc
```

Comment:
[mkrzero/variant.cpp](https://github.com/arduino/ArduinoCore-samd/blob/master/variants/mkrzero/variant.cpp#L125-L146) has entries for a 32768Hz crystal on pins 34 and 35, but I don't think Firmata users need to access them (MKR1000 also has those entries).

So the highest pin number that needs to be addressed is 33 for the Battery voltage ADC.